components/ratings/VariousAttributesRating.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <v-container fluid>
+   <v-row align="center">
+     <span id="ClearableRating" class="mr-3">Clearable</span>
+          <v-rating
+                v-model="rating"
+                color="red lighten-3"
+                background-color="grey lighten-1"
+                large
+                clearable
+              ></v-rating>
+   </v-row>
+   <v-row align="center">
+     <span id="OpenCloseDelayRating" class="mr-3">Open-delay, close-delay</span>
+          <v-rating
+                v-model="rating1"
```

Comment:
If you use "v-model" with value "rating1" please specify it's value in script data section:
rating1: ???,